linq4j/src/main/java/org/apache/calcite/linq4j/tree/Expressions.java

```diff
@@ -71,7 +81,15 @@ public static String toString(List<? extends Node> expressions, String sep,
    * Converts an expression to Java source code.
    */
   public static String toString(Node expression) {
-    return toString(Collections.singletonList(expression), "", true);
+    return toString(expression, 0);
+  }
+
+  /**
+   * Converts an expression to Java source code.
+   * Optionally splits the generated method if the method is too large.
```

Comment:
"too large" is vague. can do better.

Reply:
Changed this to be more specific about relating to the limit.